public/scripts/indexScript.js

```diff
@@ -35,44 +35,49 @@ async function changeNumberInputs(gameIds) {
 
     game_exists = false;
 
-    if (gameInput.value && gameInput.value > 0) {
-        gameIds.forEach(gameId =>{
-            if (gameId == gameInput.value)
+    if (gameInput.value) {
+        game_id = -1;
+        game_name = "";
+        gameData.forEach(game =>{
+            if (game.name == gameInput.value) {
                 game_exists = true;
+                game_id = game.id;
+                game_name = game.name;
```

Comment:
Oef `snake_case` in Javascript is eigenlijk niet heel netjes 😂, want meestal wordt daarvoor `camelCase` gebruikt. Maarja, het is niet heel erg en hoeft ook niet van mij worden aangepast.